walkthroughs/week-4-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -15,9 +15,20 @@
 package com.google.sps;
 
 import java.util.Collection;
+import java.util Arraylist;
```

Comment:
Something is weird here because GitHub shows a red square for the `.` between `util.Arraylist`. Can you check that you don't have a strange character in the file?